CardPayments/src/main/java/com/paypal/android/cardpayments/CardClient.kt

```diff
@@ -110,6 +122,26 @@ class CardClient internal constructor(
         }
     }
 
+    /**
+     * Call this method to attach a payment source to a setup token.
+     *
+     * @param context [Context] Android context
+     * @param vaultRequest [VaultRequest] request containing details about the setup token and card to use for vaulting.
+     */
+    fun vault(context: Context, vaultRequest: VaultRequest) {
```

Comment:
We probably want analytics for vault usage right? Maybe let's make that a separate ticket & include updating this [doc](https://paypal.sharepoint.com/:x:/r/sites/BTSDK/_layouts/15/Doc.aspx?sourcedoc=%7BAAA7C353-5744-4F21-AC7D-DF85D8605817%7D&file=Northstar%20Analytics%20Events.xlsx&action=default&mobileredirect=true&cid=1b0afccd-68ab-448f-9760-60ed14df9811) with the new events we send

Reply:
Yeah that makes sense to me! I'm down to sync with iOS and add them in another ticket.